scripts/ci_checks/linux-checks/check-version.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+
+tagname=$(git tag -l)
+tagname=${tagname: -5}
```

Comment:
very fragile; suggestion:

```suggestion
tagname="$(git tag -l | tail -n1)"
```